chrome/content/privly-applications

```diff
@@ -1 +1 @@
-Subproject commit 5547688ed5c632bc52755c9b5e5f783875248e91
+Subproject commit 0cd4e24320568c685924418fb27e8e5db7ae6078
```

Comment:
It's at the same state as `gsoc-jetpack`. I haven't included the changes of `gsoc-develop` into Jetpack yet. I think `gsoc-jetpack` isn't needed anymore(?) and the changes can be put into `gsoc-develop`